modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-major.adoc

```diff
@@ -0,0 +1,359 @@
+==== Message Acknowledgment
+
+In version 2.x of this connector, the `acknowledgementMode` of the subscriber was defined at the configuration level:
+
+[source,xml,linenums]
+----
+<anypoint-mq:default-subscriber-config name="Anypoint_MQ_Default_subscriber"
+       acknowledgementMode="MANUAL" acknowledgementTimeout="120000">
+    <anypoint-mq:connection clientId="${client.id}" clientSecret="${client.secret}"/>
+</anypoint-mq:default-subscriber-config>
+----
+
+These parameters are now part of the subscriber element itself and affect only that single subscriber instance. The semantics of the acknowledgment parameters has not changed.
+Here's the equivalent configuration of the subscriber:
+
+[source,xml,linenums]
+----
+<anypoint-mq:subscriber config-ref="Anypoint_MQ_Config"
+    destination="queue"
+    acknowledgementMode="MANUAL"
+    acknowledgementTimeout="2"
+    acknowledgementTimeoutUnit="MINUTES"/>
+----
+
+==== Prefetch Mode
+
+In version 2.x of this connector, `prefetch` mode was enabled by default and you specified all related parameters at the configuration level:
+
+[source,xml,linenums]
+----
+<anypoint-mq:default-subscriber-config
+      name="Anypoint_MQ_Default_subscriber"
+      pollingTime="2000" fetchSize="5"
+      fetchTimeout="5000" frequency="2000">
+    <anypoint-mq:connection clientId="${client.id}"
+      clientSecret="${client.secret}"/>
+</anypoint-mq:default-subscriber-config>
+----
+
+Version 3.x includes a new concept called `Subscriber Type`, in which you control the `prefetch` configuration.
+
+[source,xml,linenums]
+----
+<anypoint-mq:subscriber config-ref="Anypoint_MQ_Config"
+    destination="queue">
+   <anypoint-mq:subscriber-type>
+     <anypoint-mq:prefetch maxBufferSize="20"/>
+   </anypoint-mq:subscriber-type>
+</anypoint-mq:subscriber>
+----
+
+This table describes the `prefetch` parameter differences.
+
+[%header,cols="20a,30a,20a,30a"]
+|===
+| Version 2.x Parameter |Description| Version 3.x Equivalent |Description
+|`fetchSize`| Declared at Config level, controls the size of the local message buffer, which is calculated as three times the `fetchSize`.
+|`maxBufferSize`| Declared at the `Subscriber` level, explicitly controls the maximum size that the local message buffer can have. This parameter serves as a target number, attempting to fetch as many messages as needed to fill it completely.
+
+|`fetchTimeout`| Controlled the time waiting for a response from the service.
+|Deprecated| This parameter is no longer available. The connector always uses the maximum long-polling value.
+
+|`frequency`| Controlled when to recheck for messages after the queue is empty.
+|Deprecated | This parameter is no longer available. The connector now handles the frequency internally - fixed at 1 second.
+
+|`pollingTime`| Used only when `fetchSize` was `0`, effectively disabling the `prefetch` mode and moving to a `polling` mode.
+|Deprecated| This parameter is replaced by the more powerful `polling` Subscriber Type.
```

Comment:
Why is Subscriber Type capitalized? If this is a name, give it a common noun object.